src/Adapters/BlobAdapter.ts

```diff
@@ -206,17 +245,18 @@ export default class BlobAdapter implements IBlobAdapter {
         const adapterMethodSandbox = new AdapterMethodSandbox(
             this.blobAuthService
         );
+
         return await adapterMethodSandbox.safelyFetchData(async (token) => {
+            const headers = {
+                authorization: 'Bearer ' + token,
+                'Content-Type': 'application/json',
+                'x-ms-version': '2017-11-09'
+            };
             try {
                 const filesData = await axios({
                     method: 'GET',
-                    url: `${this.blobProxyServerPath}/${this.containerName}`,
-                    headers: {
-                        authorization: 'Bearer ' + token,
-                        'Content-Type': 'application/json',
-                        'x-ms-version': '2017-11-09',
-                        'x-blob-host': this.storageAccountHostName
-                    },
+                    url: this.generateBlobUrl(''),
```

Comment:
Seeing this with empty string makes me think that we should change the name of the method to something like generateBlobUrlWithPath or something.
```suggestion
                    url: this.generateBlobUrlWithPath(''),
```

Reply:
I like the naming for generateBlobUrl better. I think the naming in the argument (path: string) is enough to give people understanding of the intention of the function.